tests/test_feedexport.py

```diff
@@ -421,11 +421,12 @@ def test_parse_settings(self):
         except ImportError:
             raise unittest.SkipTest("GCSFeedStorage requires google-cloud-storage")
 
-        settings = {'GCS_PROJECT_ID': '123', 'FEED_STORAGE_GCS_ACL': 'publicRead'}
+        settings = {'GCS_PROJECT_ID': '123', 'FEED_STORAGE_GCS_ACL': 'publicRead', 'timeout': 60}
```

Comment:
The option is not called `timeout`, so I would expect this to fail provided we put a value other than the default value.